### ISSUE TITLE
IGroupForm breaks Organization Tab Navigation

### ISSUE THREAD

**Issue by FabiApfelkern** [ckan/ckan#1399]
I’m using the IGroupForm in CKAN 2.1.1 for customizing the Organization forms (group_types return organization). This causes that the „Datasets“ tab in the organization view is not working correctly anymore. It is not highlighted when active and there is no icon. I narrowed it down to the 
line _menu_items = config['routes.named_routes'] in helpers.py.
It seems that the reverse routing is not working correctly. In _*menu*_items the key „organization_read“ is set to:
{'action': 'read', 'needed': ['id'], 'controller': 'group',  'highlight_actions': 'read', 'icon': None}

So somehow the controller is wrong. I have no idea what causes this behaviour. The string organization_read occurs only three times in my project. But still somewhere something changes that value.

To reproduce that error, you just have to implement the IGroupForm in an extension.

**Comment by smihai** [comment]
Solved this by adding method to my custom **OrganizationForm** class:
```
    def group_controller(self):
        return 'organization'
```

**Comment by wardi** [comment]
Fixed by ec8ea872
